synapse/config/experimental.py

```diff
@@ -127,6 +127,9 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
             "msc3886_endpoint", None
         )
 
+        # MSC3930: Push rules for MSC3391 polls
+        self.msc3930_enabled: bool = experimental.get("msc3930_enabled", False)
```

Comment:
Will polls need anything else? I don't think servers care about MSC3381, but would make it sense to name it `polls_enabled`?

Reply:
I don't think anything else will be needed, no. So we may as well say this is enabling server-side support for polls. The only danger of calling it strictly `polls_enabled` is if another polls MSC comes around, however unlikely that is.

But just for precedence's sake, how does `msc3381_polls_enabled` sound?

Reply:
In the past we had done things like this, e.g. we had `spaces_enabled`. I'm OK with either (or even not changing it!) but wanted to throw the thought out!

Reply:
I think the more user-friendly name would definitely attract more people to turn it on and try it out. Even with the comment I'm not sure normal people will turn this on currently even if they want to use polls, unless they were specifically told to do so.

Let's change it and see what happens :)